curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --distractors--
 
-Placeholder distractor 1
+`ls <text> >> <filename>`
 
 ---
 
-Placeholder distractor 2
+`cp <text> >> <filename>`
 
 ---
 
-Placeholder distractor 3
+`mv <text> >> <filename>`
 
 #### --answer--
 
-Placeholder answer
+`echo <text> >> <filename>`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command is used to remove directories and their content recursively?
 
 #### --distractors--
 
-Placeholder distractor 1
+`cd -r`
 
 ---
 
-Placeholder distractor 2
+`cp -r`
 
 ---
 
-Placeholder distractor 3
+`dir -r`
 
 #### --answer--
 
-Placeholder answer
+`rm -r`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command is used to remove a complete directory?
 
 #### --distractors--
 
-Placeholder distractor 1
+`rm -r`
 
 ---
 
-Placeholder distractor 2
+`rm`
 
 ---
 
-Placeholder distractor 3
+`rm --help`
 
 #### --answer--
 
-Placeholder answer
+`rmdir`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command is used to go back to the previous directory?
 
 #### --distractors--
 
-Placeholder distractor 1
+`mv ..`
 
 ---
 
-Placeholder distractor 2
+`dir ..`
 
 ---
 
-Placeholder distractor 3
+`cp ..`
 
 #### --answer--
 
-Placeholder answer
+`cd ..`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which flag can be used for any command to get a help menu?
 
 #### --distractors--
 
-Placeholder distractor 1
+`--info`
 
 ---
 
-Placeholder distractor 2
+`--r`
 
 ---
 
-Placeholder distractor 3
+`-l`
 
 #### --answer--
 
-Placeholder answer
+`--help`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command is used to close the terminal?
 
 #### --distractors--
 
-Placeholder distractor 1
+`close`
 
 ---
 
-Placeholder distractor 2
+`mv`
 
 ---
 
-Placeholder distractor 3
+`off`
 
 #### --answer--
 
-Placeholder answer
+`exit`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command is used to clear the terminal?
 
 #### --distractors--
 
-Placeholder distractor 1
+`clr`
 
 ---
 
-Placeholder distractor 2
+`clean`
 
 ---
 
-Placeholder distractor 3
+`clrscr`
 
 #### --answer--
 
-Placeholder answer
+`clear`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command is used to copy the whole folder?
 
 #### --distractors--
 
-Placeholder distractor 1
+`dir -r <folder_to_copy> <name_of_copy>`
 
 ---
 
-Placeholder distractor 2
+`mv -r <folder_to_copy> <name_of_copy>`
 
 ---
 
-Placeholder distractor 3
+`cd -r <folder_to_copy> <name_of_copy>`
 
 #### --answer--
 
-Placeholder answer
+`cp -r <folder_to_copy> <name_of_copy>`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which flag is used with find to search for exact folder?
 
 #### --distractors--
 
-Placeholder distractor 1
+`cd -name <folder>`
 
 ---
 
-Placeholder distractor 2
+`ls -name <folder>`
 
 ---
 
-Placeholder distractor 3
+`mv -name <folder>`
 
 #### --answer--
 
-Placeholder answer
+`find -name <folder>`
```

Comment:
These answers don't align with the question. The question asks about the flag, not the command.